src/app/Models/Setting.php

```diff
@@ -11,4 +11,31 @@ class Setting extends Model
 
     protected $table = 'settings';
     protected $fillable = ['value'];
+
+    public static function get($key)
```

Comment:
The lack of PHP doc is disturbin.

Reply:
Totally agree. At the time I thought "_heck, they're just getters and setters_". But now, let's do this right :-) Thank you for keeping me in check. Added in https://github.com/Laravel-Backpack/Settings/commit/3064341962c9018df725ac17fb2defee24f7eef7